hack/tests/sanity-check.sh

```diff
@@ -7,7 +7,8 @@ go fmt ./...
 
 ./hack/check_license.sh
 ./hack/check_error_log_msg_format.sh
-./hack/doc/gen_cli_doc.sh
+./hack/generate/gen_cli_doc.sh
+./hack/generate/gen-test-framework.sh
```

Comment:
Nit: can we follow the same convention for underscores or dashes in the filenames? Looks like we're using underscores for other script names.

Reply:
I notice that all other scripts were using - and not _. So, I change what was required to keep all using _ instead of. 

<img width="441" alt="Screenshot 2019-11-12 at 15 23 38" src="https://user-images.githubusercontent.com/7708031/68684576-9e5b3c00-0560-11ea-91e0-bb8379989917.png">